reflex/utils/exec.py

```diff
@@ -81,9 +88,17 @@ def run_process_and_launch_url(run_command: list[str]):
 
     while True:
         if process is None:
+            kwargs = {}
+            if constants.IS_WINDOWS:
+                kwargs["creationflags"] = subprocess.CREATE_NEW_PROCESS_GROUP
```

Comment:
```suggestion
                kwargs["creationflags"] = subprocess.CREATE_NEW_PROCESS_GROUP  # type: ignore
```

This field doesn't exist on posix systems, so add type ignore to appease pyright.

When the static checks are passing we can merge it.